src/main/java/com/vinylteam/vinyl/entity/Currency.java

```diff
@@ -33,6 +40,9 @@ public String getSymbol() {
                 || currencyDescription.equals("&nbsp;€")) {
             resultingCurrency = EUR;
         }
+        logger.debug("Resulting optional with currency is {'Optional.ofNullable(resultingCurrency)':{}}",
+                Optional.ofNullable(resultingCurrency));
```

Comment:
What info we can see in file log after Optional.ofNullable(resultingCurrency).toString() ? and how we can use this info during debugging process?